modules.d/95udev-rules/module-setup.sh

```diff
@@ -57,9 +56,21 @@ install() {
     # eudev rules
     inst_rules 80-drivers-modprobe.rules
 
-    if dracut_module_included "systemd"; then
-        inst_multiple -o ${systemdutildir}/network/*.link
-        [[ $hostonly ]] && inst_multiple -H -o /etc/systemd/network/*.link
+    # legacy persistent network device names versus systemd predictable names
+    # Never use both schemes at the same time!
+    if dracut_module_included "network"; then
+        if [ -z $systemdnetwork ]; then
+            systemdnetwork="/usr/lib/systemd/network"
+        fi
+        if [ -e /etc/udev/rules.d/70-persistent-net.rules ]; then
+            if [ -e "$systemdnetwork"/99-default.link ]; then
+                mv "$systemdnetwork"/99-default.link \
+                   "$systemdnetwork"/99-default.link_disabled
+            fi
+            inst_rules 70-persistent-net.rules
+        elif [ -e "$systemdnetwork"/99-default.link ]; then
+            inst_rules "$systemdnetwork"/99-default.link
```

Comment:
`99-default.link` is not a udev rule, so `inst_rules` should not be used here, but `inst_simple`.